apple/repositories.bzl

```diff
@@ -99,12 +99,10 @@ def apple_rules_dependencies(ignore_version_differences = False):
     _maybe(
         http_archive,
         name = "bazel_skylib",
-        patches = ["@build_bazel_rules_apple//doc:bazel-skylib.pr307.patch"],
-        patch_args = ["-p1"],
         urls = [
-            "https://github.com/bazelbuild/bazel-skylib/releases/download/1.0.3/bazel-skylib-1.0.3.tar.gz",
+            "https://github.com/bazelbuild/bazel-skylib/archive/df3c9e2735f02a7fe8cd80db4db00fec8e13d25f.tar.gz",
         ],
-        sha256 = "1c531376ac7e5a180e0237938a2536de0c54d93f5c278634818e0efc952dd56c",
+        sha256 = "58f558d04a936cade1d4744d12661317e51f6a21e3dd7c50b96dc14f3fa3b87d",
```

Comment:
`strip_prefix` is needed for GitHub's auto-generated tarballs.

Reply:
thanks!